### ISSUE TITLE
Running branch for minor versions

### ISSUE THREAD

**Issue by standage** [dib-lab/khmer#1710]
This afternoon @ctb suggested that we create a version 2.1 branch from which we can cut any subsequent necessary bugfix releases. The idea is that we want to get on with merging #1595 and other stuff without having to wait for any changes that may result from the JOSS review.

**Comment by ctb** [comment]
Yes - I suggest branch '2.1_maint' for continued work on 2.1, and then
switch master over to a wild free-for-all of cython.

On Wed, May 24, 2017 at 03:16:14PM -0700, Daniel Standage wrote:
> This afternoon @ctb suggested that we create a version 2.1 branch from which we can cut any subsequent necessary bugfix releases. The idea is that we want to get on with merging #1595 and other stuff without having to wait for any changes that may result from the JOSS review.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/dib-lab/khmer/issues/1710
-- 
C. Titus Brown, ctbrown@ucdavis.edu

**Comment by betatim** [comment]
Sounds like a good plan. We should remember to cherry pick changes to `2.1_maint` and apply them to `master`.
